zeppelin-interpreter-parent/pom.xml

```diff
@@ -128,13 +128,13 @@
         </configuration>
       </plugin>
 
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-dependency-plugin</artifactId>
-        <configuration>
-          <skip>true</skip>
-        </configuration>
-      </plugin>
+      <!--<plugin>-->
+        <!--<groupId>org.apache.maven.plugins</groupId>-->
+        <!--<artifactId>maven-dependency-plugin</artifactId>-->
+        <!--<configuration>-->
+          <!--<skip>true</skip>-->
+        <!--</configuration>-->
+      <!--</plugin>-->
```

Comment:
Could you uncomment this ? It is my mistake in my PR.

Reply:
I have not modified this pom.xml.

Reply:
This is due to my PR

Reply:
done.